vendor.js issues

- [x] add `vendorScripts` to build (instead of just serve)
 - [x] test whether vendor.js gets built and copied properly

Created a simple test case for this. It seems to be creating the vendor.js and including it correctly now, but when being served, is throwing error below.

```
Uncaught Error: Cannot find module 'function r(n,t,e){function u(i,a){if(!t[i]){if(!n[i]){var f="function"==typeof require&&require;if(!a&&f)return f(i,!0);if(o)return o(i,!0);var c=new Error("Cannot find module '"+i+"'");throw c.code="MODULE_NOT_FOUND",c}var l=t[i]={exports:{}};n[i][0].call(l.exports,function(r){var t=n[i][1][r];return u(t?t:r)},l,l.exports,r,n,t,e)}return t[i].exports}for(var o="function"==typeof require&&require,i=0;i<e.length;i++)u(e[i]);return u}'
```

This reminds me of the concat error we saw previously when mapbox.js was missing a ';'?

--- a/app/index.html
+++ b/app/index.html
@@ -44,8 +44,10 @@
       ga('create','UA-XXXXX-X');ga('send','pageview');
     </script>
 
-    <!-- build:js assets/scripts/bundle.js -->
+    <!-- build:js assets/scripts/vendor.js -->
     <script src="assets/scripts/vendor.js"></script>
+    <!-- endbuild -->
+    <!-- build:js assets/scripts/bundle.js -->
     <script src="assets/scripts/bundle.js"></script>
     <!-- endbuild -->
 </body>

